pages/home.md

```diff
@@ -14,14 +14,14 @@ permalink: /
         />
       </figure>
       <p class="subtitle has-text-centered is-size-4 is-size-3-tablet is-size-2-desktop">
-      Using Web standards to
-      let people control their data, and choose the
-      applications and services to use with it.
+        Developing Web standards that empower people to take control of their
+        data, including choosing how your data is used.
```

Comment:
The current suggestion switches from third person to second person. I also think it's quite verb-heavy (developing / empower / take / choosing / used).

Completely different suggestion, more as a tagline:

```suggestion
        Solid: your data, your choices.<br>
        Empowering people via Web standards.
```

Since all the rest of the info is mentioned below. And "Solid" wasn't on the front page yet.

Reply:
BTW we need to be consistent in Web or web. I prefer the former, but I don't mind either way. It's just written differently below.

Reply:
100% agree that the suggestion became way too verb-heavy and just long in general.  And I'm totally pro-tagline - but I wonder whether we need to explicitly state that the solid is a project that's developing web the standards?  It's kind of implied through "via Web standards" but it would be nice (maybe?) if a reader  could have get a of what the solid project is, instead of just the desired results(maybe?)

Reply:
Oh - good catch on Web vs. web.  Will do a search on the site so that we're consistent.

Reply:
> Empowering people by advancing Web standards.

or any other verb?

Reply:
that works.

Reply:
Resolving this as we've continued the thread via https://github.com/solid/solidproject.org/pull/475#discussion_r555289643.